pkg/controller/constraint/constraint_controller.go

```diff
@@ -609,35 +631,55 @@ func (c *ConstraintsCache) reportTotalConstraints(ctx context.Context, reporter
 	}
 }
 
-func IsVapAPIEnabled() bool {
+func IsVapAPIEnabled() (bool, schema.GroupVersion) {
 	vapMux.Lock()
 	defer vapMux.Unlock()
 
 	if VapAPIEnabled != nil {
-		return *VapAPIEnabled
+		return *VapAPIEnabled, GroupVersion
```

Comment:
Who sets GroupVersion? what if it's never set?

Reply:
`GroupVersion` is set alongside `VapAPIEnabled=true` whenever VAP api with `v1` or `v1beta` is found. If its never set, `GroupVersion` will be `nil`, but in that case `*VapAPIEnabled` will always be `false`.

Reply:
since GroupVersion can be nil, please check for nil for the returned value from the caller function